.github/workflows/gauntlet.yml

```diff
@@ -26,9 +24,10 @@ jobs:
           nix develop -c yarn install --frozen-lockfile
         env:
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
Please also remove the `NPM_TOKEN` setup from workflows, as `gautlet-core` should now be public so this shouldn't be necessary anymore.